tests/expressions/tests.py

```diff
@@ -2849,3 +2849,34 @@ def test_nulls_false(self):
             F("field").asc(nulls_first=False)
         with self.assertRaisesMessage(ValueError, msg):
             F("field").desc(nulls_last=False)
+
+
+def test_order_by_with_outerref_comprehensive(self):
```

Comment:
This looks like pytest style, not unittest style. Django's tests follow unittest style (using a class with instance methods).

```py
tests % ./runtests.py -k test_order_by_with_outerref_comprehensive
Testing against Django installed in '/Users/.../django/django' with up to 10 processes
Found 0 test(s).
System check identified no issues (17 silenced).

----------------------------------------------------------------------
Ran 0 tests in 0.000s

NO TESTS RAN
```